src/pages/dashboard/Dashboard.tsx

```diff
@@ -44,30 +38,18 @@ const Dashboard: FC = () => {
 
   return (
     <StyledDashboard>
+      <StyledAbout>
+        <Card>
```

Comment:
smør på flesk eller? trenger vel ikke begge? 
```suggestion
      <StyledBackgroundDiv>
```

Reply:
Kan godt gjøre det
![bilde](https://user-images.githubusercontent.com/6313468/105957650-d7a29d80-6079-11eb-9d91-062691cdef31.png)

La bare inn Card for en enklest mulig 1. versjon, for Kine kom jo med nye skisser til hjem i går. Tenker vi kan lage en ny oppgave som fikser hjem som nye skisser når de er "godkjent"

Reply:
må fikse linker i dark mode også ser jeg